docs/architecture/adr-006-non-interactive-defaults.md

```diff
@@ -108,47 +108,49 @@ Note: In order to properly test the new encoding scheme, we have to perform iden
 
 We currently utilize a struct to store the state needed for lazily writing message shares. Here we add a method to it that allows for us to write namespaced padded shares.
 
+<!-- markdownlint-disable MD010 -->
```

Comment:
[Non Blocking][Question]
Why disabling this? All other ADRs are using spaces instead of tabs. Is there a specific reason for this? 
I guess if we no longer want spaces instead of tabs, we could disable the markdown rule, better. What do you think?

Reply:
yeah, second this

Reply:
> Why disabling this?

Because Go fmt uses tabs.

**Indentation**
We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

Ref: https://go.dev/doc/effective_go#formatting

If this rule isn't disabled, markdownlint will complain for correctly formatted Go code blocks and `markdownlint --fix` will reformat tabs to spaces (in other words change properly formatted Go code into unformatted Go code).

> All other ADRs, as far as I know, are using spaces instead of tabs.

True. This isn't for an ADR but there are other instances of disabling this rule prior to Go code blocks: 
- https://github.com/celestiaorg/celestia-app/blob/8e35ab2d06e38e7c71059b4f26c45b1944162ba7/x/payment/spec/docs.md?plain=1#L26
- https://github.com/celestiaorg/celestia-app/blob/8e35ab2d06e38e7c71059b4f26c45b1944162ba7/x/payment/spec/docs.md?plain=1#L118

> I guess if we no longer want spaces instead of tabs, we could disable the markdown rule from the linter directly.

I think we want this rule for all Markdown text _except_ code blocks and fortunately there is a config param for this. 

Ref: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md010---hard-tabs

Reply:
Resolved in https://github.com/celestiaorg/celestia-app/pull/874/commits/54777cefffb620d7aa0cd8b5f67d795d09c1b1f3